<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>titleWid</class>
 <widget class="QWidget" name="titleWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

#titleWid{
background-color:  rgba(185, 185, 185,125);
color: rgba(185, 185, 185,125);
border.width:0
}

 #frameCtrl, #frameSystem{
 background-color: transparent;
}

#minimizeButton {
  image: url(&quot;:/images/minimum@3x.png&quot;);
  border-radius: 3px;
}


#minimizeButton, #maximizeButton {
  background-color:transparent;
  border-radius: 3px;
}

#maximizeButton {
  image: url(&quot;:/images/maximum@3x.png&quot;);
  
}

#maximizeButton[max=&quot;false&quot;] {
  image: url(&quot;:/images/maximum@3x.png&quot;);
  
}

#maximizeButton[max=&quot;true&quot;] {
  image: url(&quot;:/images/maxOut@3x.png&quot;);
  
}

#minimizeButton:hover, #maximizeButton:hover {
  border-style: none;
  background-color: #70b7b7c7;
}

#minimizeButton:pressed, #maximizeButton:pressed {
  border-style: none;
  background-color: #80808080;
}

#maximizeButton:pressed {
   
   background-color: #80808080;
}
#closeButton {
  image: url(&quot;:/images/closeWin@3x.png&quot;);
  background-color: transparent;
  border-radius: 3px;
}

#closeButton:hover {
  border-style: none;
  background-color: #e81123;
}

#closeButton:pressed {
  border-style: none;
  background-color: #8c0a15;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameSystem">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>140</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="styleSheet">
         <string notr="true">font-size: 18px;
background-color: rgba(0, 0, 0, 0);
  font-weight: 500;
  text-align: center;
  color: #fff;
</string>
        </property>
        <property name="text">
         <string>AnimationGif</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimizeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>26</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>34</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="maximizeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>26</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>34</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>26</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>34</width>
          <height>26</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameCtrl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
